specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json

```diff
@@ -4829,6 +4837,18 @@
           "format": "int32",
           "description": "Specifies the fault domain of the virtual machine."
         },
+        "computerName": {
+          "type": "string",
+          "description": "The computer name assigned to the virtual machine."
+        },
+        "osName": {
+          "type": "string",
+          "description": "The Operating System running on the virtual machine."
```

Comment:
is this `readOnly`? if yes please mark it so.

Reply:
I will revert the change where I have marked them ReadOnly because it is not needed as VirtualMachineInstanceView itself is ReadOnly